lib/compiler/ops-to-jsir.ts

```diff
@@ -493,9 +493,12 @@ export default class OpCompiler {
         }
 
         let expiration_date = null;
+        let once = null;
         if (streamop.expiration_date !== null)
             expiration_date = this.compileValue(streamop.expiration_date, this._currentScope);
-        this._irBuilder.add(new JSIr.InvokeAtTimer(iterator, timeArray, expiration_date));
+        if (streamop.once !== null)
+            once = this.compileValue(streamop.once, this._currentScope);
```

Comment:
It's probably cleaner if we handle the case of streamop.once === null here and make it:
```
once = this._irBuilder.allocateRegister();
this._irBuilder.add(new JSIr.LoadConstant(once, false));
```

That way, invokeAtTimer doesn't need to handle a "null" case. Also, we should change the declaration of invokeAtTimer in ExecEnvironment accordingly.